android-bindings/lib-wrapper/android-bindings/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Nit: This seems to duplicate the `.gitignore` in the parent dir.

Reply:
addressed in the comment below